### PR TITLE
[fix]: bug while keploy test in user app

### PR DESCRIPTION
**Pull request by AhmedLotfy02**
## Related Issue
  - Info about Issue or bug

Closes: #1576

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
-Checks for the existance of the folder before granting access to it
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added
-I tested it without testcases and with testcases.
Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by AhmedLotfy02** [comment]
I have read the CLA Document and I hereby sign the CLA

File: cmd/test.go [hunk 1]
```diff
@@ -448,12 +448,14 @@ func (t *Test) GetCmd() *cobra.Command {
...
-
-				cmd := exec.Command("sudo", "chmod", "-R", "777", path)
-				err = cmd.Run()
-				if err != nil {
-					t.logger.Error("failed to set the permission of keploy directory", zap.Error(err))
-					return err
+				_, err := os.Stat(path)
+				if !os.IsNotExist(err) {
+					cmd := exec.Command("sudo", "chmod", "-R", "777", path)
+					err = cmd.Run()
```
Comment (shivamsouravjha) [inline cmd/test.go:454]: instead of directly using this use the util function as this would give uncontrolled security path error.

Comment (AhmedLotfy02) [reply cmd/test.go]: @shivamsouravjha  Done & Tested.

**Review by shivamsouravjha — APPROVED** [review]
LGTM!
image

**Pull request merged**